### PR TITLE
Setup for vuex and typescript and linting

### PR DESCRIPTION
**Pull request by JakubC-projects**

### CODE REVIEW

File: client/src/router/routes.js [hunk 1]
```diff
@@ -8,6 +9,11 @@ export const routes = [{
     name: 'Home',
     component: Home
   },
+  {
+    path: '/test',
+    name: 'Test',
```
Comment (StevenMalaihollo) [inline client/src/router/routes.js:14]: Hmm, now we have a JS and a TS file for the same thing?

I think we should remove the JS files

Comment (JakubC-projects) [reply client/src/router/routes.js:14]: Ok, I can remove them

**Review by StevenMalaihollo — APPROVED** [review]
This is good for now, we need to get this merged so we don't step on eachothers feet while developing

**Review by JakubC-projects — COMMENTED** [review]
Good job

**Pull request merged**